js/data/load_options.d.ts

```diff
@@ -1,4 +1,14 @@
+import { FilterDescriptor, GroupDescriptor, SelectDescriptor, SortDescriptor } from './index';
```

Comment:
Check if load_options.d.ts is connected with load_options.js. If not - I think it's better to move it to the data/index.d.ts

Reply:
Check how LoadOptions is used and if it can be used in all cases with descriptors.